app/interfaces/database/user_repository.go

```diff
@@ -9,6 +9,19 @@ type UserRepository struct {
 	SqlHandler
 }
 
+func (repo *UserRepository) FindIdByEmployeeId(employeeId string) (id int, err error) {
```

Comment:
This naming is a bit confusing.
I am assuming that it is the unique ID that is autogenerated by Employee ID string??

if yes, then do we need it specifically? 
Or do you jsut want to speed up by using the ID column instead of searching through the employee_id column strings??

Reply:
I changed FindIdByEmployeeId to FindKeyIdByEmployeeId

Reply:
Awesome!